### PR TITLE
[NL] hide charts if there are more than 6

### PR DESCRIPTION
**Pull request by beets**

### CODE REVIEW

File: static/css/nl_interface.scss [hunk 1]
```diff
@@ -113,20 +113,29 @@ $answer-min-height: 100vh;
...
+  .line-chart .legend,
+  .block-title {
     display: none;
   }
 
-  .block-title {
-    display: none;
+  .tile-hidden {
+    position: absolute;
+    top: -1000px;
```
Comment (chejennifer) [inline static/css/nl_interface.scss:123]: nit: could this just be `display: none`?

File: static/js/components/subject_page/block.tsx [hunk 2]
```diff
@@ -52,10 +53,16 @@ export interface BlockPropType {
...
+const NUM_TILES_SHOWN = 6; // for NL: show 2 rows (3 tiles per row).
+const HIDE_TILE_CLASS = "tile-hidden";
+
 export function Block(props: BlockPropType): JSX.Element {
   const columnWidth = props.columns
     ? `${(100 / props.columns.length).toFixed(2)}%`
     : "0";
+  // HACK for NL. Assumes all charts are in a single column.
+  const showExpando = isNlInterface();
+  const hideIfIndexGreater = showExpando ? NUM_TILES_SHOWN : -1;
```
Comment (chejennifer) [inline static/js/components/subject_page/block.tsx:65]: nit: minIdxToHide or something like that? 'hideIfIndexGreater' sounds like it should be a boolean of whether or not to hide.

File: static/js/components/subject_page/block.tsx [hunk 3]
```diff
@@ -81,13 +95,33 @@ export function Block(props: BlockPropType): JSX.Element {
...
+  const selfEl = e.target as HTMLAnchorElement;
+  const parentEl = selfEl.parentElement;
+  const tiles = parentEl.getElementsByClassName(
+    HIDE_TILE_CLASS
+  ) as HTMLCollectionOf<HTMLElement>;
+  for (let i = 0; i < tiles.length; i++) {
+    tiles[i].classList.remove(HIDE_TILE_CLASS);
+  }
+  selfEl.hidden = true;
+  e.preventDefault;
```
Comment (chejennifer) [inline static/js/components/subject_page/block.tsx:110]: e.preventDefault()

**Review by chejennifer — APPROVED** [review]
thanks, small things!

File: static/css/nl_interface.scss (hunk 1 above)
Comment (beets) [reply static/css/nl_interface.scss:123]: with display: none, all the calls to getBBox() during chart drawing receives 0, so the charts never render. this way, they are rendered but out of the way

File: static/js/components/subject_page/block.tsx (hunk 2 above)
Comment (beets) [reply static/js/components/subject_page/block.tsx:65]: good call - done

File: static/js/components/subject_page/block.tsx (hunk 3 above)
Comment (beets) [reply static/js/components/subject_page/block.tsx:110]: thanks for the catch! done

**Pull request merged**